### ISSUE TITLE
You can get the correct words from http://www.chinesetest.cn/index.do

### ISSUE THREAD

**Issue by lirata333** [elkmovie/hsk40#6]
First ， You may login in the http://www.chinesetest.cn/index.do
Then, You can find the window .click it.

**Comment by darxkies** [comment]
There is only this http://www.chinesetest.cn/standardsAction.do?means=standardInfo but no downloadable files right?

**Comment by darxkies** [comment]
It seems someone did download that files: https://github.com/shawkynasr/HSK-official-Query-System

**Comment by lirata333** [comment]
> It seems someone did download that files: https://github.com/shawkynasr/HSK-official-Query-System

Yes . Maybe it contains some errors. 
![image](https://user-images.githubusercontent.com/87461879/126075783-2bd6fd09-29ad-41bd-bb4c-6a58f81035e1.png)

**Comment by lirata333** [comment]
> There is only this http://www.chinesetest.cn/standardsAction.do?means=standardInfo but no downloadable files right?

Yes. We can get the data by  Web Crawler .  Very easy

**Comment by lirata333** [comment]
Sorry&nbsp; for my late reply. This is HSK website by official&nbsp;http://www.chinesetest.cn/standardsAction.do?means=standardInfo. There is no download files. But its the best wordlist in the internet. In your last email, you said someone in Github supply the wordlist for HSK. He said that he got the wordlist by OCR. Maybe some wrong message in his wordlist. Any question please e mail to me?




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "elkmovie/hsk30"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年7月17日(星期六) 中午1:34
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [elkmovie/hsk30] You can get the correct words from http://www.chinesetest.cn/index.do (#6)





 
There is only this http://www.chinesetest.cn/standardsAction.do?means=standardInfo but no downloadable files right?

**Comment by ofou** [comment]
> > It seems someone did download that files: https://github.com/shawkynasr/HSK-official-Query-System
> 
> Yes . Maybe it contains some errors. ![image](https://user-images.githubusercontent.com/87461879/126075783-2bd6fd09-29ad-41bd-bb4c-6a58f81035e1.png)

it seems now they're fixed.

**Comment by ofou** [comment]
Have anyone tried to get the list too? I'm getting some mistakes out of it in https://github.com/shawkynasr/HSK-official-Query-System/issues/5

**Comment by shawkynasr** [comment]
I fixed it according [official book](https://www.blcup.com/EnPInfo/index/10867#001)

[You can check now](https://github.com/shawkynasr/HSK-official-Query-System)

**Comment by mkizilkaya90** [comment]
@shawkynasr 
there are 3000 characters in official list. but yours has 3149?

**Comment by shawkynasr** [comment]
> [@shawkynasr](https://github.com/shawkynasr) there are 3000 characters in official list. but yours has 3149?

You can check the official website or books, total is 3149

**Comment by mkizilkaya90** [comment]
> > [@shawkynasr](https://github.com/shawkynasr) there are 3000 characters in official list. but yours has 3149?
> 
> You can check the official website or books, total is 3149

Yes I checked many times before replying you, there are 2 documents for version 3.0 (2021 and 2026)
There are 3000 漢字 + 1200 hanzi for handwriting in version 2021.
There are 3088 hanzi + 1200 hanzi for handwriting in version 2026.

Can you link your sources for 3149 please?

Thank you in advance!

**Comment by mkizilkaya90** [comment]
I just uploaded my version for the new HSK 3.0 2026
https://github.com/profesorm/hsk30

**Comment by shawkynasr** [comment]
> > > [@shawkynasr](https://github.com/shawkynasr) there are 3000 characters in official list. but yours has 3149?
> > 
> > 
> > You can check the official website or books, total is 3149
> 
> Yes I checked many times before replying you, there are 2 documents for version 3.0 (2021 and 2026) There are 3000 漢字 + 1200 hanzi for handwriting in version 2021. There are 3088 hanzi + 1200 hanzi for handwriting in version 2026.
> 
> Can you link your sources for 3149 please?
> 
> Thank you in advance!

Yes, there are one book “国际中文教育中文水平等级标准等级描述、音节、汉字National Standard 1 Grade descriptors syllables and Characters”

<img width="1475" height="226" alt="Image" src="https://github.com/user-attachments/assets/81fd6d19-ebc2-4014-b750-e7a4896aa260" />

**Comment by ofou** [comment]
So, which one is the most up to date? 👀

**Comment by mkizilkaya90** [comment]
> So, which one is the most up to date? 👀

https://github.com/profesorm/hsk30
